### ISSUE TITLE
Is a good thing to have Socket.prototype.runRequestQueue method

### ISSUE THREAD

**Issue by albertosouza** [balderdashy/sails.io.js#28]
Hi is a good practice to add Socket.prototype.runRequestQueue to execute requests?

I made the change on a fork and it seems to be working well. 

Link: https://github.com/albertosouza/sails.io.js/blob/master/sails.io.js#L573

and run automaticaly in : https://github.com/albertosouza/sails.io.js/blob/master/sails.io.js#L759

If it's a good idea I can try to add a setting for this feature and send a pull request.

**Comment by sailsbot** [comment]
Thanks for posting, @albertosouza.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
